bedrock/firefox/templates/firefox/new/scene1_windows.html

```diff
@@ -20,10 +20,12 @@
 {#- This will appear as <meta property="og:description"> which can be used for social share -#}
 {% block page_og_desc %}{{_('Faster page loading, less memory usage and packed with features, the new Firefox for Windows is here.')}}{% endblock %}
 
-{% block header_content %}
+{% block main_title %}
 <header>
-  <h1>{{ _('Firefox fights for you') }}</h1>
-  <h2>{{ _('on Windows.') }}</h2>
+  <h1>
+    {{ _('Firefox fights for you') }}
```

Comment:
Let's make this one string like: `{{ _('Firefox fights for you <span>on Windows.</span>') }}` and add an l10n comment above it that the span is for visual formatting... though I doubt there are any plans to localize these pages.